notebook/static-src/services/contents.js

```diff
@@ -252,5 +249,6 @@ define(function(require) {
         return this.get(path, {type: 'directory'});
     };
 
-    return {'Contents': Contents};
-});
+    exports.Contents = Contents;
+    
+    window.define([], function() { return {Contents: Contents}; })
```

Comment:
Ahah :smiling_imp: 

That makes this dynamically loadable using requirejs.  I wonder if I actually need it now that I'm AMDifying everything.  I'll try removing it.